mercury/views/simulator.py

```diff
@@ -13,40 +25,91 @@ class SimulatorView(TemplateView):
     def post(self, request, *args, **kwargs):
         """Used by AJAX method in the simulator.js file to save data
         from the simulator UI."""
-        post_created_at = request.POST.get("created_at")
-        post_temperature = request.POST.get("temperature")
-        post_acceleration_x = request.POST.get("acceleration_x")
-        post_acceleration_y = request.POST.get("acceleration_y")
-        post_acceleration_z = request.POST.get("acceleration_z")
-        post_wheel_speed_fr = request.POST.get("wheel_speed_fr")
-        post_wheel_speed_fl = request.POST.get("wheel_speed_fl")
-        post_wheel_speed_br = request.POST.get("wheel_speed_br")
-        post_wheel_speed_bl = request.POST.get("wheel_speed_bl")
-        post_suspension_fr = request.POST.get("suspension_fr")
-        post_suspension_fl = request.POST.get("suspension_fl")
-        post_suspension_br = request.POST.get("suspension_br")
-        post_suspension_bl = request.POST.get("suspension_bl")
-        post_current_fuel_level = request.POST.get("current_fuel_level")
-        sim_data = SimulatedData(
-            created_at=post_created_at,
-            temperature=post_temperature,
-            acceleration_x=post_acceleration_x,
-            acceleration_y=post_acceleration_y,
-            acceleration_z=post_acceleration_z,
-            wheel_speed_fr=post_wheel_speed_fr,
-            wheel_speed_fl=post_wheel_speed_fl,
-            wheel_speed_br=post_wheel_speed_br,
-            wheel_speed_bl=post_wheel_speed_bl,
-            suspension_fr=post_suspension_fr,
-            suspension_fl=post_suspension_fl,
-            suspension_br=post_suspension_br,
-            suspension_bl=post_suspension_bl,
-            current_fuel_level=post_current_fuel_level,
-        )
-        sim_data.save()
+
+        if request.POST.get("created_at_accel"):
```

Comment:
A long if/elif/elif/elif/else block like this is a code smell. I think there is a probably a much cleaner way to implement this. Perhaps we can just write to the proper API endpoint from the JavaScript directly. Let's think about it some more.